oarepo_communities/ui/oarepo_communities/templates/CommunityRecordsPage.jinja

```diff
@@ -0,0 +1,21 @@
+{#def  extra_context, search_app_config #}
+{%- set title = _("Community") ~ " " ~ request.path.split('/')[-1] ~ " | " ~ _("NR Document repository") %}
```

Comment:
also community name shouldnt be parsed from url like this, cant we use somehing better coming from context props?